sdk/src/jumbf_io.rs

```diff
@@ -12,26 +12,59 @@
 // each license.
 
 use crate::{
-    asset_handlers::{c2pa_io::C2paIO, jpeg_io::JpegIO, png_io::PngIO},
+    asset_handlers::{bmff_io::BmffIO, c2pa_io::C2paIO, jpeg_io::JpegIO, png_io::PngIO},
     asset_io::{AssetIO, CAILoader, HashObjectPositions},
     error::{Error, Result},
 };
 
 use std::{
-    fs,
+    fs::{self, File},
     io::Cursor,
     path::{Path, PathBuf},
 };
 
-static SUPPORTED_TYPES: &[&str; 6] = &[
+static SUPPORTED_TYPES: [&str; 18] = [
+    "avif",
```

Comment:
I think we have a design issue here: Some of these file formats are only available if you've enabled the `bmff` crate feature, right? Not a blocker for this PR, but I'd like to rethink how we describe what is and isn't supported in the face of multiple feature sets.

Reply:
@scouten-adobe   We don't currently have a way to advertise supported formats.  Also some may be for reading, writing,  in-memory, wasm, or other.